Add tiles_with_hash view to normalized mbtiles files (#834)
urbica/martin: docs/src/tools.md

      FROM map
      JOIN images ON images.tile_id = map.tile_id;
    ```

>   Optionally, `.mbtiles` files with this schema can also include the following view:

Reviewer: ```suggestion
  Optionally, `.mbtiles` files with `normalized` schema can include a `tiles_with_hash` view:
```
